### DISCUSSION TITLE
run functions on row create

### DISCUSSION THREAD

**Discussion by joekman990** [AntlerVC/firetable#933] [Support / Q&A] [answered]
I would love to be able to create cloud functions as they are in firebase with triggers like on create 
 and are tied to the collection

**Comment by harinij** [comment, marked as answer]
Hi @joekman990 this is already possible with Rowy :) You can use the extensions feature to create cloud functions for on create, update or delete. Here is a [quick clip](https://youtu.be/3-R5KyRh4EE?t=1340) of how you can do it and also the [documentation](https://docs.rowy.io/extensions).

Do share your feedback on Discord in case you found this feature useful or if you would like any improvements.
